src/plugins/home/public/application/components/add_data/add_data.tsx

```diff
@@ -138,18 +139,62 @@ export const AddData: FC<Props> = ({ addBasePath, application, isDarkMode, isClo
             </EuiFlexGroup>
           </EuiFlexItem>
 
-          <EuiFlexItem>
-            <EuiImage
-              alt="Illustration of Elastic data integrations"
-              className="homDataAdd__illustration"
-              src={
-                addBasePath('/plugins/kibanaReact/assets/') +
-                (isDarkMode
-                  ? 'illustration_integrations_darkmode.svg'
-                  : 'illustration_integrations_lightmode.svg')
-              }
-            />
-          </EuiFlexItem>
+          {isCloudEnabled && (
```

Comment:
I think the check we want here is actually `isCloudEnabled === false`. ECMA should only display for on-prem users, and not on cloud.